leonid_andrushchenko/src/main/java/lesson8/Group.java

```diff
@@ -2,11 +2,60 @@
 
 import java.util.ArrayList;
 
-public interface Group {
-    String getName();
-    ArrayList <Student> getStudents();
-    void addStudent(Student student);
-    void deleteStudent(String student_name);
-    boolean getStatus();//Возвращает true если группа действительна, в пративном случае возвращает false
-    Subject getSubject();// Возвращает опциональный предмет, который проходят в данной гуппе
+public class Group {
+    private String name;
+    private boolean isActual = false;// данная переменная показывает является ли группа действительной
+    private ArrayList<Student> students = new ArrayList<Student>();// Список студентов, состоящих в данной группе
+    private Subject subject; // предмет, который проходится в данной группе
+
+    public Group(String name, Subject subject){
+        this.name = name;
+        this.subject = subject;
+    }
+
+
+    public String getName(){
+        return name;
+    }
+
+
+    public ArrayList <Student> getStudents(){
+        return students;
+    }
+
+
+    public void addStudent(Student student){
```

Comment:
Please, move methods add, remove and etc into StudentService and GroupManager. These classes are responsible for logic. Group, Student,SchoolClass and Subject are models which represent data and relations between each other